app/controllers/users_controller.rb

```diff
@@ -10,7 +10,7 @@ def edit; end
   def show; end
 
   def destroy
-    authorize! :destroy, User
+    authorize! :destroy, current_user
```

Comment:
Are you sure that it's `current_user` here?

Reply:
done